LORIS_indepth.html

```diff
@@ -516,23 +179,74 @@ <h2>Survey Module</h2>
                       </section>
 
 		    </section>
-		    
-                    <!-- Statistics
-		    <section>
-		      <section>
+              
+    <!-- LORIS key features Column 4 -->
+        <section>
+            
+            <section>
+            <img src="images/LORIS_features.png">
+                 <p>Scroll down to see more key features of LORIS</p>
```

Comment:
Reveal doesn't actually work with mouse scroll - you have to hit the arrow keys.  Can we find a better word than "scroll" to clarify this?

Reply:
good idea - we don't want to use click either in case someone is viewing on a touchscreen. Maybe "See more key LORIS features below"